rero_ils/config.py

```diff
@@ -116,42 +116,54 @@ def _(x):
     ('de', _('German')),
     ('it', _('Italian')),
 ]
+# Define the default system currency in used. Each organisation can override
+# this parameter using the 'default_currency' field
+RERO_ILS_DEFAULT_CURRENCY = 'CHF'
 
 # Base templates
 # ==============
 #: Global base template.
 BASE_TEMPLATE = 'rero_ils/page.html'
-THEME_BASE_TEMPLATE = BASE_TEMPLATE
 #: Cover page base template (used for e.g. login/sign-up).
 COVER_TEMPLATE = 'rero_ils/page_cover.html'
-THEME_COVER_TEMPLATE = COVER_TEMPLATE
 #: Footer base template.
-THEME_FOOTER_TEMPLATE = 'rero_ils/footer.html'
+FOOTER_TEMPLATE = 'rero_ils/footer.html'
 #: Header base template.
 HEADER_TEMPLATE = 'rero_ils/header.html'
-#: Header base template.
-THEME_HEADER_TEMPLATE = HEADER_TEMPLATE
-#: Settings page template used for e.g. display user settings views.
-THEME_SETTINGS_TEMPLATE = 'invenio_theme/page_settings.html'
+#: Settings base template
+SETTINGS_TEMPLATE = 'invenio_theme/page_settings.html'
+#: Admin base template
+ADMIN_BASE_TEMPLATE = BASE_TEMPLATE
+
+# Miscellaneous variable around templates
+# =======================
 #: Template for security pages.
 SECURITY_LOGIN_USER_TEMPLATE = 'rero_ils/login_user.html'
 SECURITY_REGISTER_USER_TEMPLATE = 'rero_ils/register_user.html'
 SECURITY_FORGOT_PASSWORD_TEMPLATE = 'rero_ils/forgot_password.html'
 SECURITY_RESET_PASSWORD_TEMPLATE = 'rero_ils/reset_password.html'
-#: Template for error pages.
-THEME_ERROR_TEMPLATE = 'rero_ils/page_error.html'
 #: Template for tombstone page.
 RECORDS_UI_TOMBSTONE_TEMPLATE = "rero_ils/tombstone.html"
+#: Miscellaneous templates
+SEARCH_UI_JSTEMPLATE_RESULTS = (
```

Comment:
As I know we do not use invenio-search-ui anymore.

Reply:
Same as previously: to avoid potential curious behaviour with non set variables.